### PR TITLE
Updated sorted submissions function

### PR DESCRIPTION
**Pull request by gabaroar**

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -56,75 +57,49 @@ func (s Session) DefaultFrontpage() ([]*Submission, error) {
 	return submissions, nil
 }
 
-// SubredditSubmissions returns the submissions on the given subreddit.
-func (s Session) SubredditSubmissions(subreddit string) ([]*Submission, error) {
+func (s Session) SortedSubmissions(subreddit string, sort popularitySort, config map[string]string) ([]*Submission, error) {
```
Comment (jzelinskie) [inline session.go]: Can we renamed `config` to `params`? What kind of parameters are you passing here; can we do something more specific than a generic map[string]string?

Comment (gabaroar) [inline session.go]: I brought my laptop with me today to work so, I was able to update the function.  The whole thing looks like this.

    client := geddit.NewSession("golangtestbotthethird")

	var params = map[string]string{
		"t":     geddit.ThisMonth,
		"limit": "10",
	}
	submissions, _ := client.SortedSubmissions("golang", geddit.TopSubmissions, params)

	for _, submission := range submissions {
		fmt.Println(submission.Title)
	}

**Comment by jzelinskie** [comment]
Looks good, I'll merge it if you add a doc comment above the function signature.

**Comment by gabaroar** [comment]
Added a comment above the SortedSubmissions function

**Comment by jzelinskie** [comment]
thanks for the contribution!

**Pull request merged**